docs/tailwind.config.js

```diff
@@ -40,6 +41,9 @@ module.exports = {
     },
   },
   plugins: [
+    plugin(function ({ addVariant }) {
+      addVariant('peer-adjacent', ':merge(.peer) + &'); // adds custom support for adjacent sibling combinator
```

Comment:
Whoaaa, I had never seen (or used) the `:merge()` directive.

I looked it up on MDN before realising it is a new Tailwind thing 😓 

<img width="859" alt="CleanShot 2023-06-20 at 16 16 20@2x" src="https://github.com/Thinkmill/keystatic/assets/485747/d2663db7-1d6f-4a61-bd52-640bf87f7212">

Reply:
Yeah it was fun to learn a bit about and try out, but I'm still curious why they don't have a thing out-of-the-box with the adjacent sibling combinator (or have I missed something? I googled a lot over the weekend haha). Maybe it's just not that useful/common 🤔